src/app/modules/onboarding/pages/nonhd-wallet-wizard/nonhd-wallet-wizard.component.ts

```diff
@@ -148,29 +139,40 @@ export class NonhdWalletWizardComponent implements OnInit, OnDestroy {
     this.loading = true;
     // We attempt to create a wallet followed by a call to
     // signup using the wallet's password in the validator client.
-
-    const observablesToExecute: Observable<any>[] = [this.importKeystores$];
-   
-    const slashingProtectionFile = this.slashingProtection?.importedFiles[0];
-    if(slashingProtectionFile ){
-      const reqImportSlashing: ImportSlashingProtectionRequest = {
-        slashing_protection_json: JSON.stringify(slashingProtectionFile)
-      }
-      observablesToExecute.push(this.walletService.importSlashingProtection(reqImportSlashing));
-    }
-  
     // a delay after the create wallet as it is non blocking once the wallet is created.
     // the import keystores request requires a keymanager to be set which is retreived from the wallet.
     // if the import keystores request is called too soon after wallet creation the keymanager may not yet be set.
     this.walletService.createWallet(request).pipe(
       delay(2000),
       switchMap((resp) => {
-        return zip(...observablesToExecute).pipe(
-          switchMap((res) => {
+        return this.importKeystores$.pipe(
+          tap((res: ImportKeystoresResponse) => {
+            // will redirect as long as it is a 200 response.
+            // if the response is not a 200 it will be handled by the catchError.
+            // validator keys may not successfully be imported despite a 200 response.
+            // data response will be printed to console.
             if(res){
               this.router.navigate([LANDING_URL]);
+              console.log(res);
+              res.data.forEach((data:ImportKeystoresData,index:number) => {
+                let pubkey = this.pubkeys[index]??'undefined pubkey';
+                if(data.status === 'IMPORTED'){
+                  this.toastr.success(
+                    `${pubkey}... IMPORTED`,
+                  );
+                } else if ( data.status === 'DUPLICATE'){
+                  this.toastr.warning(
+                    `${pubkey}... DUPLICATE, no action taken`,
+                  );
+                }else {
+                  this.toastr.error(
+                    `${pubkey}... status: ${data.status}`,
+                    `${data.message !== ''? data.message : 'IMPORT failed'}`,{
```

Comment:
Should we say that key that it failed for?

Reply:
I'll improve this

Reply:
the first item is the title, it includes the pubkey, but I improved the message a bit below it.